crates/forge/tests/e2e/running.rs

```diff
@@ -453,11 +453,57 @@ fn init_new_project_test() {
         .arg("test_name")
         .assert()
         .success();
+    let manifest_path = temp.child("test_name/Scarb.toml");
+
+    let generated_toml = std::fs::read_to_string(&manifest_path.path()).unwrap();
+    let version = env!("CARGO_PKG_VERSION");
+    let expected_toml = formatdoc!(
+        r#"
+            [package]
+            name = "test_name"
+            version = "0.1.0"
+
+            # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
+
+            [dependencies]
+            snforge_std = {{ git = "https://github.com/foundry-rs/starknet-foundry", tag = "v{}" }}
+            starknet = "2.2.0"
+
+            [[target.starknet-contract]]
+            casm = true
+            # foo = {{ path = "vendor/foo" }}
```

Comment:
Could we get rid of that? I think we can just remove commented lines in `init`, I guess `toml_edit` should make it easy

Reply:
@mkaput told that this comment will be removed from scarb